src/main/res/values/strings.xml

```diff
@@ -1283,8 +1283,8 @@
 	<string name="image_gallery_no_image_present_message">The specified image could not be found. It may have been deleted.</string>
 
 	<!-- 2020-09-04 -->
-	<string name="reddit_login_browser_popup_line_1">To login, you will now be taken to Reddit in a web browser.</string>
-	<string name="reddit_login_browser_popup_line_2">When the login is complete, you should be redirected back to RedReader. If the login fails, you can try a different browser using the checkbox below.</string>
+	<string name="reddit_login_browser_popup_line_1">To log in, you will now be taken to Reddit in a web-browser.</string>
```

Comment:
No need for the hyphen

Reply:
Consistent with pre-fetch

Reply:
"Prefetch" and "precache" should also not be hyphenated

Reply:
To me it seems "pre-cache" is the verb, whereas "precache" is the noun. I much prefer the action, and both return hits.